JakinChan200/CS180_Roald
# #13 Setup client routes

## Overview
Adds dropdown menus onto regional pages
Adds navbar
Adds regional pages with proper linking navigation/routes
Prunes old css from starter boilerplate


## Materials
- Closes #13 


## Pictures
![navigation](https://user-images.githubusercontent.com/43893085/163284649-618329e2-dad8-4cc5-85bb-b2d5583924d9.gif)


## Manual Testing Guide
- Start the client and navigate to `localhost:3000`
- Note that there is now a navigation bar 
- Note that clicking the regions changes the page
- Note clicking the triangle buttons on the region pages hides/shows the contents below
- Note that clicking the world button on the left of the navbar returns to home

Pull requests should receive approval before merge.
